readme.md

```diff
@@ -1,6 +1,11 @@
 
 # Ignition Ewon® Connector
 
+## [NOTICE]
+
+The Ignition Ewon Connector is currently undergoing development for Version 2.0.0.
+During this time, documentation and source code may be relocated, unavailable, or incomplete.
+
```

Comment:
For major rewrites like this would it be more appropriate to have a Ignition-8-2.0.0 branch and then merge to main once complete?

Reply:
That would be the preferred path. During the master -> main rename and retiring of the Ignition 7.9 module, the v2.0.0 code got jumbled around and merged to the main branch though.

After this v2.0.0 release, I intend to use the preferred path for future instances.